docker-compose.yml

```diff
@@ -3,6 +3,7 @@ version: '3.7'
 services:
   traefik:
     image: "traefik:v2.2.1"
+    env_file: .env
```

Comment:
This is the default for docker-compose.  If you leave this line out, .env will still be loaded.